inst/examples/face-detection/app.R

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env Rscript
+
+
+## ---- Install packages ----
+
+# install.packages("opencv")
+
+
+## ---- load packages ----
+
+library(RestRserve)
+library(opencv)
+
+
+## ---- create handler for the HTTP requests ----
+
+check_request = function(request, types, limit) {
+  if (length(request$body) == 0L) {
+    raise(HTTPError$bad_request())
+  }
+  h_ctype = request$content_type
+  if (!(h_ctype %in% allowwed_types)) {
+    raise(HTTPError$unsupported_media_type())
+  }
+  if (length(request$body) > limit) {
+    raise(HTTPError$payload_too_large())
+  }
+  return(invisible(TRUE))
+}
+
+face_handler = function(request, response) {
+  allowed_types = paste("application", c("jpeg", "png"), sep = "/")
+  allowed_limit = 10 * 1024 * 1024 # 10 Mb
+  check_request(request, allowed_types, allowed_limit)
+
+  tmp = tempfile()
+  on.exit(unlink(tmp))
+  writeBin(as.raw(request$body), tmp)
+  img = ocv_read(tmp)
+  faces = ocv_facemask(img)
+  response$set_body(attr(faces, "faces"))
```

Comment:
let's add comment that `attr(faces, "faces")` is a list with `x`, `y`, `radius` members